mkosi/config.py

```diff
@@ -2134,6 +2135,12 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         parse=config_make_list_parser(delimiter=","),
         help="Add additional packages to the default initrd",
     ),
+    ConfigSetting(
+        dest="devicetree",
+        section="Content",
+        parse=config_parse_string,
```

Comment:
You probably want `config_make_path_parser(required=False)` here.

Reply:
Thanks for the hint. Applied.